app/Resources/views/layout.html.twig

```diff
@@ -122,7 +122,7 @@ along with Incipio as the file LICENSE.  If not, see <http://www.gnu.org/license
             {# It is illegal (cf licence) to remove the link to github #}
             {# And dishonest to remove the reference to Jeyser's main contributors.#}
             <p class="text-muted" style="text-align: center">
-                <i>© Jeyser CRM 2017 &#124; N7 Consulting - AB</i>
+                <i>© Jeyser CRM 2017 &#124; M-GaTE &#124; N7 Consulting</i>
```

Comment:
Very minor but IMO it would be better if it was M-Gate &  N7 Consulting rather than M-Gate | N7 Consulting.

It returns the idea of cooperation and suits more with project philosophy